raiden/tests/integration/contracts/test_network_registry.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from raiden.exceptions import (
+    TransactionThrew,
+    InvalidAddress,
+    AddressWithoutCode,
+    NoTokenManager,
+)
+from eth_utils import is_same_address, to_canonical_address
+
+
+@pytest.fixture
+def get_random_address(get_random_bytes):
+    return lambda: get_random_bytes(20)
```

Comment:
again, we don't need this pattern. please use the existing utilities:

https://github.com/raiden-network/raiden/blob/990827def9f24266ee01287940485e48aea04bf6/raiden/tests/utils/factories.py#L80-L81